resources/function_help/json/array_minority

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "array_minority",
+  "type": "function",
+  "groups": ["Arrays"],
+  "description": "Return the less common values in an array.",
+  "arguments": [
+    {"arg":"array","description":"an array"},
+    {"arg":"option='all'", "optional":true, "description":"a string specifying the return values handling. Valid options are:<br /><ul><li>all: Default, all less common values are returned in an array</li><li>first: Return the first of the less common values</li><li>median: Return the median of the less common values</li><li>real_minority: Return all values except the real majority if thre is one</li></ul>"}
+  ],
+  "examples": [
+    { "expression":"array_minority(array(0,42,42), 'all')", "returns":"[ 0 ]"},
+    { "expression":"array_minority(array(0,1,42,42), 'all')", "returns":"[ 0, 1 ]"},
+    { "expression":"array_minority(array(0,1,42,42,43,1), 'first')", "returns":"43"},
```

Comment:
This logic does not seem to match the one of array_majority. I'd expect it to return 0 (the one whose count has first reached the min).